indy_common/test/types/test_rs_schema_schema.py

```diff
@@ -0,0 +1,58 @@
+from collections import OrderedDict
+
+from indy_common.constants import RS_JSON_LD_ID, RS_JSON_LD_TYPE, TXN_TYPE
+from indy_common.types import ClientSetRsSchemaOperation, RsSchemaMetaField, \
+    ClientSetContextOperation, ContextField, RsSchemaField, SetRsSchemaDataField, ClientSetRsSchemaOperation
+from plenum.common.messages.fields import ConstantField, VersionField, IterableField, LimitedLengthStringField
+
+EXPECTED_SCHEMA_META_FIELDS = OrderedDict([
+    ("name", LimitedLengthStringField),
+    ("version", VersionField),
+    ("type", ConstantField),
+])
+
+
+def test_meta_has_expected_fields_s():
+    actual_field_names = OrderedDict(RsSchemaMetaField.schema).keys()
+    assert actual_field_names == EXPECTED_SCHEMA_META_FIELDS.keys()
+
+
+def test_meta_has_expected_validators_s():
+    schema = dict(RsSchemaMetaField.schema)
+    for field, validator in EXPECTED_SCHEMA_META_FIELDS.items():
+        assert isinstance(schema[field], validator)
+
+
+EXPECTED_RS_SCHEMA_DATA_FIELDS = OrderedDict([
+    (RS_JSON_LD_TYPE, LimitedLengthStringField),
+    (RS_JSON_LD_ID, LimitedLengthStringField),
```

Comment:
Should we expect `RS_JSON_LD_CONTEXT ` (`@context`) here as well?

Reply:
No. `@context` is optional. I hope the RFC will be updated to better reflect this. https://github.com/hyperledger/indy-hipe/blob/master/text/0149-rich-schema-schema/README.md#example-schema .
The current example in the RFC is missing the required `@id` and `@type` while including optional `@context`.